src/app.js

```diff
@@ -1 +1,10 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+fs.cp('./src/1.txt', './src/1 copy.txt', (err) => {
```

Comment:
The source and destination file paths are hardcoded. Consider using dynamic file paths for better usability and to prevent errors.